app/src/test/java/com/keylesspalace/tusky/StringUtilsTest.kt

```diff
@@ -20,6 +22,15 @@ class StringUtilsTest {
         notLessList.forEach { (l, r) -> assertFalse("not $l < $r", l.isLessThan(r)) }
     }
 
+    @Test
+    fun inc() {
+        listOf(
+                "122" to "123",
+                "12A" to "12B",
+                "1" to "2"
```

Comment:
No test case for something ending with `z`?

Reply:
or `9`?

Reply:
`inc()` is very dumb and I just let it increase any character, really. Even if it's z it should become something bigger.